.github/workflows/wheels-test.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 set -e
 
+# Ensure fribidi is installed by the system.
 if [[ "$OSTYPE" == "darwin"* ]]; then
-    brew install fribidi
-    export PKG_CONFIG_PATH="/usr/local/opt/openblas/lib/pkgconfig"
-    if [ -f /opt/homebrew/lib/libfribidi.dylib ]; then
-        sudo cp /opt/homebrew/lib/libfribidi.dylib /usr/local/lib
+    # If Homebrew is on the path during the build, it may leak into the wheels.
+    # However, we *do* need Homebrew to provide a copy of fribidi for
+    # testing purposes so that we can verify the fribidi shim works as expected.
+    if [[ "$(uname -m)" == "x86_64" ]]; then
+        # Use the "installed" location, rather than /usr/local, for two reasons:
+        # firstly, Homebrew allows libraries to be *installed*, but not linked;
+        # and secondly, we don't want any *other* leakage from /usr/local.
+        HOMEBREW_HOME=/usr/local/Homebrew
+    else
+        HOMEBREW_HOME=/opt/homebrew
     fi
+    $HOMEBREW_HOME/bin/brew install fribidi
+
+    # Add the lib folder for fribidi so that the vendored library can be found.
+    # Don't use /opt/homebrew/lib directly - use the lib folder where the
+    # installed copy of fribidi is cellared. This ensures we don't pick up the
+    # Homebrew version of any other library that we're dependent on (most notably,
+    # freetype).
+    export DYLD_LIBRARY_PATH=$(dirname $(realpath $HOMEBREW_HOME/lib/libfribidi.dylib))
```

Comment:
```suggestion
    if [[ "$(uname -m)" == "x86_64" ]]; then
        FRIBIDI_DYLIB=$(find /usr/local/Cellar/fribidi -type f -name libfribidi.dylib)
    else
        FRIBIDI_DYLIB=$HOMEBREW_HOME/lib/libfribidi.dylib
    fi
    export DYLD_LIBRARY_PATH=$(dirname $FRIBIDI_DYLIB)
```

https://github.com/python-pillow/Pillow/actions/runs/11752543578/job/32744256384?pr=8497#step:5:9142
> realpath: /usr/local/Homebrew/lib/libfribidi.dylib: No such file or directory

libfribidi.dylib isn't located there on the macOS Intel jobs.

Reply:
Huh... at some point, I've clearly done something to mess up my x86 test machine's Homebrew install, because I've got a `/usr/local/Homebrew/Cellar`... but I now see I'm also getting a bunch of warnings because of the existence of that folder. 

The `find`-based solution you've got here will fail (or be prone to failure) if there's more than one version of fribidi installed, which is possible on user machines; but if `HOMEBREW_HOME` is moved back to `/usr/local`, the `realpath` approach should work.